app/models/setting.rb

```diff
@@ -170,7 +170,7 @@ def defaults
         "hot_score_period_in_days": 31,
         "related_content_score_threshold": -0.3,
         "featured_proposals_number": 3,
-        "dashboard.emails": nil,
+        "feature.dashboard.notification_emails": nil,
```

Comment:
I think it's a good idea @javierm, but I have a question... 
Currently the key that disappears ("dashboard.emails") was being displayed in Settings > Configuration... that is, it was not an Enable/Disable key, it was a value filled with a input field.
For this reason, I would discard to use: "settings:rename_setting_keys" because even if it works, the new key would save the text introduced by the administrator with the old version, instead of True or False...
And if we only use "settings:remove_deprecated_settings", in case someone already has updated "dashboards.emails" he would lose this feature. 🤔 
How about the next solution:
We create a new task to add in the consul.rake that only removes the old key and in case it has a value we updates the new key with true? in case it has no value we leave it in false/nil

Reply:
As far as I know, keys in the "Featues" section are not stored as `true` or `false`, but either as a string or `nil`. That's because the type of the "value" column in the "settings" table is a string.

This is something I've just tried in the console:

```ruby
Setting["feature.featured_proposals"] = "aaa" # Makes the setting appear as "enabled"
Setting["feature.featured_proposals"] = "" # Makes the setting appear as "disabled"
```

So I guess renaming the key would maintain the current behaviour :thinking:.

Reply:
Ok @javierm !!!
I agree with you as "almost" always 😄 
I was a little worried about the bbdd value:
```
> Setting["feature.featured_proposals”] = true
> Setting["feature.featured_proposals”]
=> "t" 
```

```
Setting["feature.featured_proposals”] = false
> Setting["feature.featured_proposals”]
=> nil 
```
But as you rightly say with the rename_key task we will maintain the expected behavior.